lunar/distribution.yaml

```diff
@@ -1255,6 +1255,22 @@ repositories:
       url: https://github.com/orocos/rtt_ros_integration.git
       version: toolchain-2.9
     status: maintained
+  srdfdom:
+    doc:
+      type: git
+      url: https://github.com/ros-planning/srdfdom.git
+      version: kinetic-devel
+    release:
+      tags:
+        release: release/lunar/{package}/{version}
+      url: https://github.com/ros-gbp/srdfdom-release.git
+      version: 0.4.2-0
+    source:
+      test_pull_requests: true
```

Comment:
Sorry for the extra burden if any but thank you @dirk-thomas.

It's possible that the one that opens PR against rosdistro doesn't have an admin right for the repo so that the ROS buildfarm user may not be added yet by the time of running `bloom`. I [updated the wiki])http://wiki.ros.org/action/info/buildfarm/Pull%20request%20testing?action=diff&rev2=16&rev1=15) to encourage users to first take care of buildfarm user.